src/lib/services/state-service.ts

```diff
@@ -417,7 +417,7 @@ export default class StateService {
             await this.eventStore.store({
                 type: DROP_ENVIRONMENTS,
                 createdBy: userName,
-                data: { name: 'all-projects' },
+                data: { name: 'all-environments' },
```

Comment:
I've got a feeling that this was supposed to say "all-environments" all along, so I updated it. @ivarconr can you confirm that? (You were the last one to touch this line of code according to git).